peps/pep-0753.rst

```diff
@@ -0,0 +1,322 @@
+or package indices:
+
+* Packaging tooling: no changes to the correctness or well-formedness
+  of the core metadata. This PEP proposes deprecations as well as behavioral
+  refinements, but all currently (and historically) produced metadata will
+  continue to be valid per the rules of its respective version.
+* Package indices: indices will continue to expect well-formed core metadata,
+  with no behavioral changes. Indices **MAY** choose to emit warnings or
+  notifications on the presence of now-deprecated fields,
+  :ref:`per above <package-indices>`.
+
+.. _future-considerations:
+
+Future Considerations
+=====================
+
+This PEP does not stipulate or require any future metadata changes.
+
+However, per :ref:`metadata-producers` and :ref:`conventions-for-labels`,
+we identify the following potential future goals for a new major release of
+the core metadata standards:
+
+* Outright removal of support for ``Home-page`` and ``Download-URL`` in the
+  next major core metadata version. If removed, package indices and consumers
+  **MUST** reject metadata containing these fields when said metadata is of
+  the new major version.
+* Enforcement of label canonicalization. If enforced, package producers
+  **MUST** emit only canonicalized ``Project-URL`` labels when generating
+  distribution metadata, and package indices and consumers **MUST** reject
+  distributions containing non-canonicalized labels. Note: requiring
+  canonicalization merely restricts labels to lowercase text, and excludes
+  whitespace and punctuation. It does NOT restrict project URLs solely to
+  the use of "well-known" labels.
+
+These potential changes would be backwards incompatible, hence their
+inclusion only in this section. Acceptance of this PEP does NOT commit
+any future metadata revision to actually making these changes.
+
+Security Implications
+=====================
+
+This PEP does not identify any positive or negative security implications
+associated with deprecating ``Home-page`` and ``Download-URL`` or with
+label canonicalization.
+
+How To Teach This
+=================
+
+The changes in this PEP should be transparent to the majority of the packaging
+ecosystem's userbase; the primary beneficiaries of this PEP's changes are
+packaging tooling authors and index maintainers, who will be able to reduce the
+number of unique URL fields produced and checked.
+
+A small number of package maintainers may observe new warnings or notifications
+from their index of choice, should the index choose to ignore ``Home-page``
+and ``Download-URL`` as suggested. Similarly, a small number of package
+maintainers may observe that their index of choice no longer renders
+their URLs, if only present in the deprecated fields. However, no package
+maintainers should observe rejected package uploads or other breaking
+changes to packaging workflows due to this PEP's proposed changes.
+
+Maintainers who observe warnings or changes to the presentation of their
```

Comment:
I wouldn't limit the learning link to just to "[m]aintainers who observe warnings or changes".  Everyone can learn how this should work from the packaging guide! 😄